app/assets/javascripts/angular/controllers/api.js

```diff
@@ -0,0 +1,79 @@
+(function () {
+  angular.module('wtsApp', ['timer', 'truncate', 'parseNews']).controller('apiCtrl', ['$scope', '$http', '$interval', '$timeout', function($scope, $http, $interval, $timeout) {
```

Comment:
New API controller works for both Desktop and Mobile. There is still a Desktop or Mobile query param that goes into it, but I may simply remove that since there is currently no difference, and it's probably better that way.